created blog

Scripore opened the pull request.

Scripore committed:
created blog

nanoxd commented on Gemfile:
```diff
@@ -0,0 +1,25 @@
+# For "summary"-Helper
+gem 'nokogiri', '~> 1.6.6.2'
+
+
```
Remove extra lines

nanoxd commented on README.md:
````diff
@@ -0,0 +1,38 @@
+
+Enter in the following commands
+```bash
+$ bundle install
````
No need to enter `$` when using backticks; causes issues with copy and pasting.

nanoxd commented on README.md:
````diff
@@ -0,0 +1,38 @@
+
+Once finished, while still in the source branch, enter the following:
+```bash
+$ git add .
````
Same as above

nanoxd commented on README.md:
````diff
@@ -0,0 +1,38 @@
+Once finished, while still in the source branch, enter the following:
+```bash
+$ git add .
+$ git commit -m '<enter in your commit message here>'
````
Avoid using the shorthand commit flag `-m`. It causes users to elongate the message. See [here](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) for more info

nanoxd commented on README.md:
````diff
@@ -0,0 +1,38 @@
+$ git push
+```
+
+Once the changes are pushed to your repo, open a pull request to submit changes to main Operation Code blog repo.
````
``` diff
- submit changes to main Operation Code blog
+ submit changes to the Operation Code blog
```

nanoxd commented on config.rb:
```diff
@@ -0,0 +1,157 @@
+  # blog.calendar_template = "calendar.html"
+
+  # Enable pagination
+  blog.paginate = true
```
Pagination is enabled but there is no hard `per_page` setting. What is the default?

nanoxd commented on config.rb:
```diff
@@ -0,0 +1,157 @@
+
+set :casper, {
+  blog: {
+    url: 'http://www.example.com',
```
URL is not set

nanoxd commented on config.rb:
```diff
@@ -0,0 +1,157 @@
+
+# Change Compass configuration
+# compass_config do |config|
+#   config.output_style = :compact
```
We should minify the CSS if it doesn't default to doing that.

nanoxd commented on config.rb:
```diff
@@ -0,0 +1,157 @@
+# Build-specific configuration
+configure :build do
+  # For example, change the Compass output style for deployment
+  # activate :minify_css
```
Enable this

nanoxd commented on config.rb:
```diff
@@ -0,0 +1,157 @@
+  # activate :minify_css
+
+  # Minify Javascript on build
+  # activate :minify_javascript
```
And this

Scripore committed:
removed extra lines and $.

nanoxd commented on helpers/middleman_casper_helpers.rb:
```diff
@@ -0,0 +1,120 @@
+  def is_tag_page?
+    current_resource.metadata[:locals]['page_type'] == 'tag'
+  end
+  def tags?(article = current_article)
```
Spacing between above method

nanoxd commented on helpers/middleman_casper_helpers.rb:
```diff
@@ -0,0 +1,120 @@
+  def tags?(article = current_article)
+    article.tags.present?
+  end
+  def tags(article = current_article, separator = ', ')
```
Same

nanoxd commented on helpers/middleman_casper_helpers.rb:
```diff
@@ -0,0 +1,120 @@
+      { class: 'no-cover' }
+    end
+  end
+  def cover?(page = current_page)
```
Same

nanoxd commented on helpers/middleman_casper_helpers.rb:
```diff
@@ -0,0 +1,120 @@
+    md5 = Digest::MD5.hexdigest(blog_author.gravatar_email.downcase)
+    "https://www.gravatar.com/avatar/#{md5}?size=#{size}"
+  end
+  def gravatar?
```
Same

nanoxd commented on helpers/middleman_casper_helpers.rb:
```diff
@@ -0,0 +1,120 @@
+    "https://twitter.com/share?text=#{current_article.title}" \
+      "&amp;url=#{current_article_url}"
+  end
+  def facebook_url
```
Same

nanoxd commented on helpers/middleman_casper_helpers.rb:
```diff
@@ -0,0 +1,120 @@
+  def facebook_url
+    "https://www.facebook.com/sharer/sharer.php?u=#{current_article_url}"
+  end
+  def google_plus_url
```
Ditto

nanoxd commented on helpers/middleman_casper_helpers.rb:
```diff
@@ -0,0 +1,120 @@
+      "#{blog.options.prefix.to_s}/feed.xml"
+    end
+  end
+  def home_path
```
Ditto

nanoxd commented on helpers/middleman_casper_helpers.rb:
```diff
@@ -0,0 +1,120 @@
+  def home_path
+    "#{blog.options.prefix.to_s}/"
+  end
+  def author_path
```
Ditto

nanoxd commented:
I made a few comments. Please fix it and squash your commits into one commit with a little more detail.

Scripore committed:
set URL to operationcode.github.io. activated minify_css

Scripore commented:
Will do!

davidmolina commented:
@Scripore looking good so far!

Scripore committed:
set per_page to 10, minify CSS and JS, add spacing

- in config.rb, uncomment minify
- in config.rb, uncomment per_page = 10
- remove -m commit flag from README.md

Scripore committed:
remove two test articles
